### PR TITLE
Feature/newsletter form

### PR DESCRIPTION
**Pull request by KGrandiek**
Update newsletter form to a more accessable one

### CODE REVIEW

**Review by bosworthco — COMMENTED** [review]
Looks good! Pulled it down and it's working for me!

- Can you please take a look at the mobile breakpoint and see if you can tighten this up? You may need to `display: none` the `<br />` as its causing whitespace (unless thats intentional).
- Make sure you stick to the naming conventions on those form classes, rather than using the random ones it comes with.
- The error/success message on mobile is very very small.
- Can you please supply a list of all the possible messages and copy and supply them to Inga in the Channel? The client will probably want a say in the copy for these.

Mobile:
image

**Comment by KGrandiek** [comment]
@bosworthco There isn't a mobile design with the error message, because it's 14px on desktop I went with the small styling which is 10px on mobile. The error/success messages come from the package so I'm not sure how many there are or how we can change them, there seem to be a couple different ones for error status.

File: components/subscribe.js [hunk 1]
```diff
@@ -1,8 +1,74 @@
...
+        <input
+          value={email}
+          type="email"
+          onChange={e => setEmail(e.target.value)}
+          placeholder="Email"
+        />
+        <br />
+        <button type="submit">Submit</button>
+
+        {status === "sending" && <div className="msg-alert">sending...</div>}
```
Comment (bosworthco) [inline components/subscribe.js:54]: @KGrandiek We have control of these messages?

File: styles/components/subscribe.scss [hunk 2]
```diff
@@ -76,15 +76,13 @@
   }
 
   .msg-alert {
```
Comment (bosworthco) [inline styles/components/subscribe.scss:78]: Can we match the naming conventions on this class?

**Comment by bosworthco** [comment]
> There isn't a mobile design with the error message, because it's 14px on desktop I went with the small styling which is 10px on mobile.

Okay cool. Maybe we should be using the Sans Serif font here then because it's barely legible using the body font! See what you can do

**Pull request merged**